### DISCUSSION TITLE
Building package: mingw-w64-clang

### DISCUSSION THREAD

**Discussion by DavidHVernon** [msys2/MINGW-packages#9681] [Q&A]
For reasons that are not ever worth going into, I am trying to rebuild the mingw-w64-clang package with clang 13 rc4. I have never built a package before in my life so I am stumbling in the dark.  

Following the very good instructions on this repo, my first step was to rebuild the clang package with no changes. Like so...

`makepkg-mingw -sLf --skippgpcheck
`

When I do this, I get the following output...

```
==>  MINGW_ARCH: mingw64
  -> Building mingw64...
==> Making package: mingw-w64-clang 12.0.1-5 (Wed, Sep 29, 2021  2:51:48 PM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found llvm-project-12.0.1.src.tar.xz
  -> Found llvm-project-12.0.1.src.tar.xz.sig
  -> Found 0001-Use-posix-style-path-separators-with-MinGW.patch
... 45 similar lines removed ...
==> WARNING: Skipping verification of source file PGP signatures.
==> Validating source files with sha256sums...
    llvm-project-12.0.1.src.tar.xz ... Passed
    llvm-project-12.0.1.src.tar.xz.sig ... Skipped
    0001-Use-posix-style-path-separators-with-MinGW.patch ... Passed
... 45 similar lines removed ...
==> Extracting sources...
==> Starting prepare()...
```
And there  are no artifacts in the pkg directory. 

So in other words it just stops. Is there a next step? Did I miss something in the instructions?

**Comment by jeremyd2019** [comment]
That should be right.  I have previously run into issues with `--log` so maybe try without that?  I personally use
`MINGW_ARCH=X time makepkg-mingw --noconfirm --noprogressbar --syncdeps --rmdeps --nocheck --cleanbuild -f`

(`--skippgpcheck` isn't necessary if you pre-fetch the keys, which can be done with `../.ci/fetch-validpgpkeys.sh`)
